js/product/product.js

```diff
@@ -0,0 +1,21 @@
+angular.module("modelProduct", [])
+    .factory("localModelProduct", function(){
+
+    function Product(nazwa, cena_netto, cena_brutto){
+        this.nazwa = nazwa;
+        this.cena_netto = cena_netto;
+        this.cena_brutto = cena_brutto;
```

Comment:
Dotyczy to wszystkich tzn "danych pochodnych",  ktore mozna wyliczyc - takich danych sie raczej nie zapisuje zeby uniknac redundancji.